src/pages/availability-panel.tsx

```diff
@@ -21,8 +22,9 @@ const AvailabilityPanel: FC = () => {
         const { requestPromise } = fetch({ url });
         const fetchPromise = await requestPromise;
         const { tasks } = fetchPromise.data;
-        const unassigned = tasks.filter(
-          (item: task) => item.status.toLowerCase() === 'unassigned' && item.type === 'feature',
+        const unassigned = updateTasksStatus(tasks).filter(
+          (item: task) => (
+            (item.status === 'AVAILABLE') && (item.type === 'feature')),
```

Comment:
Could we not create a constant for `feature` and utilise the const `AVAILABLE` here?

Reply:
We should use here 👍